src/main/views/home.njk

```diff
@@ -3,39 +3,928 @@
     {{ title }}
```

Comment:
Ok really minor, but any chance you could remove the extra spacing in here between a lot of the sections?

Reply:
sorted.